lib/Doctrine/ORM/Configuration.php

```diff
@@ -182,17 +183,26 @@ public function newDefaultAnnotationDriver($paths = [], $useSimpleAnnotationRead
             );
         }
 
-        AnnotationRegistry::registerFile(__DIR__ . '/Mapping/Driver/DoctrineAnnotations.php');
+        if (method_exists(AnnotationRegistry::class, 'registerFile')) {
+            AnnotationRegistry::registerFile(__DIR__ . '/Mapping/Driver/DoctrineAnnotations.php');
+        }
```

Comment:
This line could be removed instead, couldn't it? https://github.com/doctrine/annotations/pull/271 ?

Reply:
I guess we could've removed it earlier already. 🤔

Reply:
Did a separate PR for that: https://github.com/doctrine/orm/pull/10321